frontend/src/components/DatePicker.js

```diff
@@ -0,0 +1,21 @@
+import React, {Fragment, useEffect, useState} from "react";
+import {DatePicker} from "@material-ui/pickers";
+
+export default function BasicDatePicker(props) {
+    let [selectedDate, handleDateChange] = useState(new Date());
```

Comment:
right now you store current date inside App state and inside DatePicker state.
We can just pass date and date change handler to DatePicker and delete local state inside DatePicker

Reply:
Fixed.